k8s/scripts/sysbox-deploy-k8s.sh

```diff
@@ -76,6 +76,7 @@ subgid_file="${host_etc}/subgid"
 
 # Shiftfs
 shiftfs_min_kernel_ver=5.4
+shiftfs_max_kernel_ver=5.18
```

Comment:
This constant needs to be updated to account for the fact that there are newer kernels than 5.18 with proper support for `shiftfs`.  We can set the new `shiftfs_max_kernel_ver` to `6.2`.

You may wonder: why 6.2 if Sysbox doesn't require `shiftfs` anymore? That's true, Sysbox doesn't require it, but we still recommend it since `id-mapped mounts` support for various file-systems isn't up to par with `shiftfs`. For example, id-mapped didn't offer `overlayfs` support till 5.19, and support for `tmpfs` just came in 6.3, which may break certain k8s features (seen an example [here](https://github.com/nestybox/sysbox/issues/728)).